.env.docker

```
@@ -48,3 +48,18 @@ DC_PPL_ENVIRONMENT=testing
 
 #Meilisearch container
 DC_MEILI_MASTER_KEY=MEILI_MASTER_KEY_FOR_DEVELOPMENT_ONLY
+
+# Payload CMS Admin Redirect
+DC_PAYLOAD_API_KEY=xxx
+DC_PAYLOAD_BASE_URL=http://localhost:8083
+DC_PAYLOAD_IFRAME_URL=http://localhost:8083
+DC_PAYLOAD_SSO_PRIVATE_KEY=
+DC_PAYLOAD_SSO_PUBLIC_KEY=
+DC_PAYLOAD_SSO_USER_EMAIL=
+DC_PAYLOAD_SSO_ISSUER=medusa
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DC_PAYLOAD_SSO_ISSUER key should go before the DC_PAYLOAD_SSO_PRIVATE_KEY key

(UnorderedKey)